nautobot_floor_plan/models.py

```diff
@@ -79,6 +92,8 @@ def get_svg(self, *, user, base_url):
 
     def save(self, *args, **kwargs):
         """Override save in order to update any existing tiles."""
+        validate_not_zero(self.x_axis_step)
```

Comment:
This should be done as a validator, like the `MinValueValidator` you have above.

Reply:
Added   validators=[validate_not_zero] to both x_axis_step and y_axis_step